src/main/java/com/cellar/stickerify/bot/StickerifyBot.java

```diff
@@ -33,9 +33,7 @@ public String getBotUsername() {
 	}
 
 	@Override
-	public String getBotToken() {
-		return System.getenv("STICKERIFY_TOKEN");
-	}
+	public String getBotToken() { return System.getenv("STICKERIFY_TOKEN"); }
```

Comment:
Hi, this was an unneeded change.
You can revert it back with the following `git` commands:

1. `git checkout origin/main src/main/java/com/cellar/stickerify/bot/StickerifyBot.java`
2. `git commit -m "reverted changes to StickerifyBot"`
3. `git push`